iree/compiler/Conversion/LinalgToSPIRV/Passes.cpp

```diff
@@ -209,18 +209,6 @@ void buildSPIRVTransformPassPipeline(OpPassManager &pm,
         Shape::createMaterializeShapeCalculationsPass());
     pm.nest<ModuleOp>().addNestedPass<FuncOp>(
         Shape::createHoistShapeCalculationsPass());
-
```

Comment:
Why we keep the shape related stuff untouched in the `else` branch? Can't we delete the `else` branch entirely?

Reply:
Let me take a stab at it. I thought some un-cleanup lit tests rely on these things. I only verified that there are no mhlo ops in lit tests, so it is totally safe to delete the pipeline.

Reply:
ah, it also works.